### DISCUSSION TITLE
Reframework for Silent Hill 2 Remake?

### DISCUSSION THREAD

**Discussion by MuhammadJA87** [praydog/REFramework#1076] [General Questions] [answered]
Are you planning to add reframework support for SH2 Remake? First person view, etc...?

**Comment by praydog** [comment, marked as answer]
No. It's not even the same engine.

My other project, UEVR is for Unreal Engine games.

**Comment by MuhammadJA87** [comment]
Do you plan to mod SH2 Remake?
